### PR TITLE
refactor: update eslint configuration + run eslint

### LINKED ISSUE #3152: [Refactor][Config-UI] Run ESLint to apply new Configuration
### Config-UI / ESLint + Prettier

As a result of PR #2995, we'll need to run ESLint against the `config-ui` package so that the new rules are enforced by the linter.

- [ ] Run ESLint
- [ ] Resolve Errors
- [ ] Fix GitHub CI Frontend-Lint Check

### PR DESCRIPTION
**Pull request by e2corporation**
### 📦 Config-UI / ESLint + Prettier

- [x] Update ESLint & Prettier Configuration
  - Define additional `react/jsx` rules for ESLint
  - Lower Prettier `printWidth` to `80` to prevent multi-line JSX collapsing
- [x] Run ESLint on `@config-ui`
- [x] Resolve Lint Errors & Warnings
- [x] `Fix` Github CI `frontend-lint` task
- [x] `Refactor` Delete Legacy V11 Artifacts 
- [x] Test Linting

### Description

This PR applies the recent **ESLint & Prettier** Configuration changes to the UI codebase, along with updating the ESLint and Prettier configuration rules for improved compatibility with JSX and React formatting.

Screen Shot 2022-09-20 at 8 06 50 PM

Screen Shot 2022-09-20 at 8 08 41 PM

### Does this close any open issues?
Closes #3152

### CODE REVIEW

File: config-ui/.prettierrc.js [hunk 1]
```diff
@@ -17,9 +17,15 @@
  */
 
 module.exports = {
-  printWidth: 140,
+  printWidth: 80,
   singleQuote: true,
+  singleAttributePerLine: false,
```
Comment (mintsweet) [inline config-ui/.prettierrc.js:22]: false should be the default value of this property, is it not necessary to add this?

Comment (e2corporation) [reply config-ui/.prettierrc.js:22]: It's defined explicitly for better clarity.

File: config-ui/.eslintrc [hunk 2]
```diff
@@ -49,8 +55,22 @@
   "rules": {
     "prettier/prettier": ["warn"],
     "camelcase": "off",
+    "semi": ["error", "never"],
     "max-len": [1, 140, 2],
```
[the remark this answers is not in the document]
Comment (e2corporation) [reply config-ui/.eslintrc:59]: I may adjust it later, right now it's not of much consequence because Prettier's printWidth will override this value.

**Comment by e2corporation** [comment]
@mintsweet PR #2995 was added too quickly to the codebase, Prettier has strong opinions on rules and was designed to have less configurable options. It also has many conflicts with ESLint with regards to how JSX is formatted, especially with multi-line blocks.

File: config-ui/src/App.js [hunk 3]
```diff
@@ -35,17 +32,11 @@ import '@fontsource/inter/variable-full.css'
...
-// import Configure from './pages/configure/index'
 import Integration from '@/pages/configure/integration/index'
 import ManageIntegration from '@/pages/configure/integration/manage'
 import AddConnection from '@/pages/configure/connections/AddConnection'
-// import EditConnection from '@/pages/configure/connections/EditConnection'
 import ConfigureConnection from '@/pages/configure/connections/ConfigureConnection'
-// import Triggers from '@/pages/triggers/index'
 import Offline from '@/pages/offline/index'
-// import Pipelines from '@/pages/pipelines/index'
-// import CreatePipeline from '@/pages/pipelines/create'
```
Comment (likyh) [inline config-ui/src/App.js:47]: Is css import in `App.js`?

Comment (e2corporation) [reply config-ui/src/App.js:47]: @likyh Any issues with css missing imports was already resolved under another PR, these imports were already disabled and commented out.

**Comment by mintsweet** [comment]
@e2corporation I'm sorry I didn't remind you about eslint going to fix errors when webpack starts up, this is an option that can be turned off, sorry again.

**Comment by e2corporation** [comment]
> @e2corporation I'm sorry I didn't remind you about eslint going to fix errors when webpack starts up, this is an option that can be turned off, sorry again.

No worries, I was hoping to get my DORA Blueprint PR merged first before the prettier eslint changes because I was trying to avoid the 150+ extra changes being adopted on my ticket, which would created an excessive diff. Once the ESLint configuration is modified it affects the global codebase so I was hoping to delay all of this refactoring till later.

**Comment by mintsweet** [comment]
> > @e2corporation I'm sorry I didn't remind you about eslint going to fix errors when webpack starts up, this is an option that can be turned off, sorry again.
> 
> No worries, I was hoping to get my DORA Blueprint PR merged first before the prettier eslint changes because I was trying to avoid the 150+ extra changes being adopted on my ticket, which would created an excessive diff. Once the ESLint configuration is modified it affects the global codebase so I was hoping to delay all of this refactoring till later.

Of course, I think that's fine, thanks.

File: config-ui/src/App.js (hunk 3 above)
Comment (e2corporation) [reply config-ui/src/App.js:47]: @likyh If you detect any new issues with missing imports let me know, these changes should be safe for now however.

**Comment by mintsweet** [comment]
LGTM.

**Review by klesh — APPROVED** [review]

**Pull request merged**